app/index.js

```diff
@@ -191,4 +201,11 @@ function handleCertificateError() {
 		config: config
 	};
 	certificateModule.onAppCertificateError(arg, logger);
+}
+
+async function requestCameraAccess() {
+	let status = systemPreferences.getMediaAccessStatus('camera');
+	logger.debug(`mac camera status ${status}`);
+	const permission = await systemPreferences.askForMediaAccess('camera');
+	logger.debug(`mac camera permission ${permission}`);
```

Comment:
maybe we can change it to something like this:
```suggestion
async function requestMediaAccess() {
	['camera','microphone','screen'].map(async (permission) => {
		const status= await systemPreferences.askForMediaAccess(permission);
		logger.debug(`mac permission ${permission} asked current status ${status}`);
	});
}
```
I haven't test it so it might not work... but you probably get the idea.